tracing-futures/Cargo.toml

```diff
@@ -31,8 +31,8 @@ futures-task = { version = "0.3", optional = true }
 pin-project = { version = "0.4", optional = true }
 tracing = "0.1"
 tokio-executor = { version = "0.1", optional = true }
-tokio = { version = "0.1", optional = true }
-tokio_02 = { package = "tokio", version = "0.2.0-alpha.6", optional = true }
+tokio_01 = { package = "tokio", version = "0.1", optional = true }
+tokio = { version = "0.2.1", optional = true }
```

Comment:
This can probably just be `0.2`, unless we specifically need something from `0.2.1` (which it doesn't look like).